src/app/Unauthenticated.tsx

```diff
@@ -2,19 +2,12 @@ import Login from 'pages/Login';
 import { Route, Routes } from 'react-router';
 
 export const logoutRoutes = {
-    path: 'logout',
-    params: {
-        reason: 'reason',
-    },
+    path: '',
```

Comment:
Redid the reason thing cause it was a major pain and kinda weird to make the login page a notification controller.